### DISCUSSION TITLE
Available ip in prefixes

### DISCUSSION THREAD

**Discussion by Sivolen** [netbox-community/netbox#8148] [Q&A]
Hi! 
Help please, 
I have updated to version 3.1.2 and can not view the available ip in prefixes. How can I see the available ip now?

**Comment by jeremystretch** [comment]
Looks like bug #8131. This has been addressed and will be fixed in v3.1.3. Sorry for the inconvenience.

**Reply by Sivolen** [reply]
Ok, thank
